### ISSUE TITLE
instantiationChannelConfiguration

### ISSUE THREAD

**Issue by afred** [WGBH/pbucore#5]

**Comment by metacirque** [comment]
'Audio track configuration' data property ('string') for Media Resource object
http://www.ebu.ch/metadata/ontologies/ebucore/dataproperties/audioTrackConfiguration___131485010.html

Seems a likely equivalent, but I am working through all of the audio elements before posting xml. 

Thoughts?

**Comment by JPEvain** [comment]
Yes, we call it audiotrackconfiguration. It can be a one stereo track configuration. Then the track is of stereo type with two channels.

It can be more more complex arrangement like define in EBU R123. In this case a track can be mono, another stereo, another surround with as many channel as required each. The precise description of channels is defined in EBUCore’s audioFormatextended, which RDF representation will be simplified.

If there is a difference that would be in the naming: EBUCore says trackconfiguration when PBCore says channelconfiguration. (but then SMPTE would speak of sound fields and sound field groups)…
